src/listener.py

```diff
@@ -0,0 +1,190 @@
+import select
+import socket
+from pascy.l2 import *
+import sys
+
+MY_ADDRESSES = {
+	"net1": ["02:42:37:94:2b:e4", "1.1.1.1"],
+	"net2": ["02:42:04:89:4c:a0", "2.2.2.1"]
```

Comment:
It is recommended to add a comma (`,`) at the end of any list item, including the last one.
This way, when someone will add support to a new protocol, the git blame won't be on him for adding the comma, but only for the relevant line.